kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/activity/ConversationActivity.java

```diff
@@ -276,6 +276,7 @@ protected void onResume() {
 
     protected void syncMessages() {
         SyncCallService.getInstance(this).syncMessages(null);
+        SyncCallService.getInstance(this).syncMessageMetadata();
```

Comment:
This call might be expensive to serve. @surajkukkipady18 you can consider this while testing, check how much time this call is taking compared to message sync.

Reply:
Have compared the speed, the difference is small. Have attached screenshots in test ticket.